inc/endpoints/class-components-endpoint.php

```diff
@@ -366,7 +375,7 @@ public function build_query() {
 		$wp_query = apply_filters( 'wp_irving_components_wp_query', $wp_query, $this->path, $this->custom_params, $this->params );
 
 		// Map to main query and set up globals.
-		// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+		// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited, WordPress.WP.GlobalVariablesOverride.OverrideProhibited
```

Comment:
Did the sniff rule change, or do we really need to be ignoring both here?

Reply:
I'm not sure I know enough about each of those vars to make an educated decision on whether or not they should be present. I do believe we should be mimicking WordPress core as much as possible though.

One of those phpcs rules hits me locally, one hits me on Travis, so I included both for the sake of it right now.